### ISSUE TITLE
projectrain with comments

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#104]
e.g. GRCpool gets projectrain but divides it onto all users, instead of dividing "only" to the users who crunched the specific WUs, because:
"right, but when a trasaction is received, i am not that i can tell what project it goes too, when i look at my personal wallet with project rain, i see nothing about a project it is tied to"

I assume either the rainmakers forget to add a comment for the rain ?

**Comment by Erkan-Yilmaz** [comment]
cc @grctest and invited scalextrix on steemit

**Comment by Erkan-Yilmaz** [comment]
see also [this comment](https://cryptocointalk.com/topic/1331-new-coin-launch-announcement-grc-gridcoin/page-1244#entry237058)

**Comment by grctest** [comment]
Yeah, I'm pretty sure that this message/label is just a note for the rain-maker's wallet. No message makes it to the end user, that would be pretty neat if it did.

**Comment by grctest** [comment]
https://github.com/gridcoin/Gridcoin-Research/issues/61

**Comment by startailcoon** [comment]
Sending rain in the Windows Client with "Project Rain" does generate a message in the TX field "hashboinc".

**Comment by startailcoon** [comment]
When sending a rain transaction from the wallet it does produce a message in the transaction of what the rain was ment for.

[This transaction for example](https://gridcoinstats.eu/tx/42074c10d0ceb4cf470e57c2bf085bf185d00a774fc978101b1a48c7f68593d1) has the following data.

> {
>     "txid" : "42074c10d0ceb4cf470e57c2bf085bf185d00a774fc978101b1a48c7f68593d1",
>     "version" : 1,
>     "time" : 1491713062,
>     "locktime" : 0,
>     "hashboinc" : "Project Rain: Project Rain: asteroids@home (Amount 100 GRC )",
>     "vin" : [....],
>     "vout" : [....],
>     "blockhash" : "a3692500fbb9a7b93afde132dbeed759baaed4b9fca380ba0e54caf6dbd8047f",
>     "confirmations" : 1
> }

It's not possible to do a sendmany with a comment however. Just as mentioned by Katiee in [gridcoin/Gridcoin-Research#61](https://github.com/gridcoin/Gridcoin-Research/issues/61)

Having a note in the sendmany command would be <i>nice</i>
